src/test/java/io/vertx/test/core/LocalSharedDataTest.java

```diff
@@ -35,6 +35,30 @@ public void setUp() throws Exception {
     sharedData = vertx.sharedData();
   }
 
+  @Test
+  public void deleteElementOnComputeFunctionReturningNull() {
+    LocalMap<String, String> map = sharedData.getLocalMap("foo");
+
+    // put an initial value
+    map.put("hello", "world");
+
+    // retuning null we should remove the entry
+    map.computeIfPresent("hello", (key, oldValue) -> null);
+    assertNull(map.get("hello"));
```

Comment:
we need instead use `contains` and not `get`

Reply:
Done